pkg/daemon/daemon.go

```diff
@@ -124,6 +125,9 @@ const (
 	pathDevNull = "/dev/null"
 	// pathStateJSON is where we store temporary state across config changes
 	pathStateJSON = "/etc/machine-config-daemon/state.json"
+	// currentConfigFingerprint is where we store the current config on disk to validate
+	// against annotations changes
+	currentConfigFingerprint = "/etc/machine-config-daemon/current"
```

Comment:
This isn't a fingerprint, it's the whole config right?  I find this idea that we serialize all of the files back into JSON as a file in the filesystem kind of funny.  But I guess it's not a huge amount of data...

```
oc get -o yaml machineconfigs/rendered-worker-74e29c89eb37d7aaa30c33c481fae174 | gzip | wc -c
10197
```

We can certainly assume 10k of data is free.

On the kind-of bikeshed topic, I think `/var` is probably a better place for this.  The ostree model is that `/etc` is data humans *might* edit (e.g. "ssh to node and `vi`").

Reply:
fixed naming and moved under `/var/machine-config-daemon`